.rubocop.yml

```diff
@@ -1,6 +1,9 @@
 AllCops:
   NewCops: enable
 
+Gemspec/RequiredRubyVersion:
+  Enabled: false
```

Comment:
I checked some documents:
https://github.com/rubocop/rubocop/blob/master/config/default.yml#L134C2-L145C49
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Gemspec/RequiredRubyVersion

It seems that TargetRubyVersion in rubocop.yml must be specified as 2.5.9 because 2.5.9 is the lowest `required_ruby_version`.
However we want rubocop to work with ruby 3.x, so I think disabling `Gemspec/RequiredRubyVersion` is proper.